db/seeds.rb

```diff
@@ -30,6 +30,20 @@
     password: "theforcevita")
 user.update(role: SiteCoordinatorRole.create(site: first_site)) if user.role_type != SiteCoordinatorRole::TYPE
 
+# site team member user
+user = User.where(email: "melon@example.com").first_or_initialize
+user.update(
+    name: "Tim Melon",
```

Comment:
Consider changing all the names in this file to be alliterative. In this case, "Marty Melon" would fit the bill. 100% not a blocker.